app/views/helpers/BackLinkHelper.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.helpers
+
+import controllers.navigation.Navigator
+import controllers.section4.routes.NatureOfTransactionController
+import controllers.summary.routes.SectionSummaryController
+import forms.section4.Document
+import models.DeclarationType
+import models.requests.JourneyRequest
+
+object BackLinkHelper {
```

Comment:
As we already have a `PreviousDocumentsHelper` class I would move this `object` in that file as
```
object PreviousDocumentsHelper {
  def backLinkText...
}
```

This way it would be more clear what page this method is for, as at the moment this is not evident.